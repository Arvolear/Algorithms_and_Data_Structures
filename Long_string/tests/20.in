4568
46139381044665556027771307753589393241447