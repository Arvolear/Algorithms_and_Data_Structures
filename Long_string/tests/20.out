46139381044665556027771307753589393246015
46139381044665556027771307753589393236879
210764692612032259934859333818396348326929896
10100565027291058675081284534498553686